docker/README.md

```diff
@@ -10,6 +10,59 @@
 </div>
 <!-- markdownlint-enable no-inline-html line-length -->
 
+## Known Issue for FiftyOne Teams v1.6.0 and Above
+
+### "Install Fiftyone" Instructions Missing PyPI Token
+
+FiftyOne Teams v1.6 introduces the Central Authentication Service (CAS), which
+introduces an abstraction layer between FiftyOne Teams and Auth0.  This
+abstraction layer makes it possible to deploy FiftyOne Teams without using Auth0
+as an Identity Service Provider.
+
+However, metadata that used to be provided to FiftyOne Teams by Auth0 is no
+longer available; which has resulted in an incomplete set of instructions in the
+[Install FiftyOne](https://docs.voxel51.com/teams/installation.html#python-sdk)
+instructions for bash.
+
+Specifically, you will see the word `TOKEN` where your Voxel51 PyPI token used
+to appear.
+
+While Voxel51 works to address this issue, you can override the install
+instructions by setting the `FIFTYONE_APP_INSTALL_FIFTYONE_OVERRIDE` environment
+value for the `teams-app` service.  This can be accomplished by adding
+something like the following to your `compose.override.yaml`:
+
+```yaml
+teams-app:
+  environment:
+    FIFTYONE_APP_INSTALL_FIFTYONE_OVERRIDE: pip install -U --index-url https://<your PyPI Token>@pypi.fiftyone.ai fiftyone==0.17.0
+```
+
+If you need your PyPI token, please contact your Customer Success representative
+and they will provide it to you.
+
+### Invitations Disabled for Internal Authentication Mode
+
+FiftyOne Teams v1.6 introduces the Central Authentication Service (CAS), which
+includes both
+[`legacy` authentication mode][legacy-auth-mode]
+and
+[`internal` authentication mode][internal-auth-mode].
+
+Inviting users to join your FiftyOne Teams instance is not currently supported
+when `FIFTYONE_AUTH_MODE` is set to `internal`.
```

Comment:
Is there an additional sentence we can add here to enable the "everyone can automatically join as a Guest" mode? I'm not sure what the right phrasing is for that.

Reply:
I don't know how "Autojoin on Authentication" works in Internal mode...  I believe that just may be the default behavior, so I'm really not sure how to document it...

Reply:
Hmmm, that's a good point. I wonder if the answer is that they autojoin and need to write a JS hook to prevent certain users from signing in or something. Either way, we can probably leave this for now and pick up the thread elsewhere.